src/components/Dialogs/Dialogs.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import style from './Dialogs.module.css'
+import { NavLink } from 'react-router-dom';
+
+const DialogItem = (props) => {
+    let path = '/dialogs/' + props.id;
```

Comment:
Consider NOT using let for variables that are NOT changing in the current scope.

Reply:
/fxd

Reply:
And? Nothing has changed here.